Doc/library/os.path.rst

```diff
@@ -488,6 +488,41 @@ the :mod:`glob` module.)
       Accepts a :term:`path-like object`.
 
 
+.. function:: splitroot(path)
+
+   Split the pathname *path* into a triad ``(drive, root, tail)`` where:
+
+   1. *drive* is a device name, mount point or the empty string;
+   2. *root* is a sequence of separators following the drive or the empty string; and
```

Comment:
I don't like the word "sequence" here — my immediate reaction is "what kind of sequence? A list? A tuple? A deque?" That said, I'm struggling to think of something better :(

Reply:
perhaps "a string of separators"?

Reply:
```suggestion
   2. *root* is either one or more separators that follow the drive or the
      empty string; and
```

Reply:
Or perhaps:

```suggestion
   2. *root* is either a string of one or more separators that follow the drive, or
      the empty string; and
```